website/docs/d/auth_backends.html.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: "vault"
+page_title: "Vault: vault_auth_backends data source"
+sidebar_current: "docs-vault-datasource-auth-backends"
+description: |-
+  List Auth Backends from Vault
+---
+
+# vault\_auth\_backends
+
+## Example Usage
+
+```hcl
+data "vault_auth_backends" "example" {}
+```
+
+```hcl
+data "vault_auth_backends" "example-filter" {
+  type = "kubernetes"
+}
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace of the target resource.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+  *Available only for Vault Enterprise*.
+
+* `type` - (Optional) The name of the auth method type.
```

Comment:
This describes what this field is but could we provide a little information on what this does?

Reply:
Yes

Reply:
I added a few more words - LMK if this is to your liking ;)